src/PlusDataCollection/NDICAPITracking/vtkPlusNDITracker.cxx

```diff
@@ -990,6 +1008,11 @@ PlusStatus vtkPlusNDITracker::ReadConfiguration(vtkXMLDataElement* rootConfigEle
 
   XML_READ_SCALAR_ATTRIBUTE_OPTIONAL(unsigned long, SerialPort, deviceConfig);
   XML_READ_SCALAR_ATTRIBUTE_OPTIONAL(unsigned long, BaudRate, deviceConfig);
+  if (vtkPlusNDITracker::ConvertBaudToNDIEnum(this->BaudRate) == -1)

```

Comment:
I added this:
  if(deviceConfig->GetAttribute("BaudRate") != NULL && vtkPlusNDITracker::ConvertBaudToNDIEnum(this->BaudRate) == -1)
  {

Reply:
Do you want me to amend my commit to add that?

Reply:
Yes please, removes a warning when no baud rate is specified.